distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc

```diff
@@ -126,25 +133,18 @@ wrapper.java.additional.13=-Dcom.sun.management.jmxremote
 wrapper.java.additional.14=-Djava.security.egd=file:/dev/./urandom
 wrapper.java.additional.15=-Dfile.encoding=UTF8
 wrapper.java.additional.16=-Dkaraf.instances=%KARAF_HOME%/instances
-wrapper.java.additional.17=-D${ddf-branding-lowercase}.home=%${ddf-branding}_HOME%
```

Comment:
maybe I'm missing it but we do we still need to define ddf.home? That was .17 previously.